### PR TITLE
Remove deprecated ioutil

### PR DESCRIPTION
**Pull request by marioferh**
fix: remove deprecated ioutil.ReadAll()

### CODE REVIEW

File: test/e2e/user_workload_monitoring_test.go [hunk 1]
```diff
@@ -122,7 +117,7 @@ func TestUserWorkloadMonitoringAlerting(t *testing.T) {
 		name string
 		f    func(*testing.T)
 	}{
-		{
+		/*{
```
Comment (slashpai) [inline test/e2e/user_workload_monitoring_test.go:120]: was this meant to be commented?

**Review by simonpasquier — APPROVED** [review]
/lgtm

File: test/e2e/user_workload_monitoring_test.go (hunk 1 above)
Comment (marioferh) [reply test/e2e/user_workload_monitoring_test.go]: sorry wrong change from other commit

**Review by slashpai — APPROVED** [review]
/lgtm

**Pull request merged**